vividus-plugin-mongodb/src/main/java/org/vividus/bdd/converter/StringToBsonConverter.java

```diff
@@ -16,22 +16,18 @@
 
 package org.vividus.bdd.converter;
 
-import java.lang.reflect.Type;
-
 import javax.inject.Named;
 
 import org.apache.commons.lang3.StringUtils;
 import org.bson.Document;
 import org.bson.conversions.Bson;
-import org.jbehave.core.steps.ParameterConverters.AbstractParameterConverter;
+import org.jbehave.core.steps.ParameterConverters.FunctionalParameterConverter;
 
 @Named
-public class StringToBsonConverter extends AbstractParameterConverter<Bson>
+public class StringToBsonConverter extends FunctionalParameterConverter<Bson>
 {
-    @Override
-    public Bson convertValue(String value, Type type)
+    public StringToBsonConverter()
     {
-        String json = StringUtils.isNoneBlank(value) ? value : "{}";
-        return Document.parse(json);
+        super(value -> Document.parse(StringUtils.isNoneBlank(value) ? value : "{}"));
```

Comment:
It is out of scope, but could you please use StringUtils. defaultIfBlank instead explicit logic?

Reply:
Oh, its noneBlank, please ignore

Reply:
I believe `isNoneBlank ` was used mistakenly

Reply:
Just checked, works for whitespaces too, so it's a valid comment :)